__tests__/app.test.js

```diff
@@ -158,4 +158,60 @@ describe("/api/reviews/:review_id/comments", () => {
     .then(({body})=>{
       expect(body.comments).toEqual([])
     })
+    
 });
+
+describe('METHOD: POST', () => {
```

Comment:
I'd add two more tests, one that ignore any unnecessary properties on an objects.
and one if the expected properties are missing from the object.